pkg/server/authentication.go

```diff
@@ -72,15 +74,15 @@ func newAuthenticationServer(s *Server) *authenticationServer {
 
 // RegisterService registers the GRPC service.
 func (s *authenticationServer) RegisterService(g *grpc.Server) {
-	serverpb.RegisterAuthenticationServer(g, s)
+	serverpb.RegisterLogInServer(g, s)
 }
 
 // RegisterGateway starts the gateway (i.e. reverse proxy) that proxies HTTP requests
 // to the appropriate gRPC endpoints.
 func (s *authenticationServer) RegisterGateway(
 	ctx context.Context, mux *gwruntime.ServeMux, conn *grpc.ClientConn,
 ) error {
-	return serverpb.RegisterAuthenticationHandler(ctx, mux, conn)
+	return serverpb.RegisterLogInHandler(ctx, mux, conn)
```

Comment:
Note to self: do I need to register both log in and log out handlers here? Strangely, log out seems to still work with just this.